### PR TITLE
Replace site with hugo

### PR DESCRIPTION
**Pull request by 602p**
- [ ] CI
 - [ ] Silver gendoc integration

Live version at https://louis.members.acm.umn.edu/www/melt-website-hugo/

### CODE REVIEW

**Comment by krame505** [comment]
Nice!  Looks much better than the current site.  

One minor thing, the dashed underline when hovering over/clicking on menu items looks kind of weird.  Is this easy to change?

**Comment by 602p** [comment]
I'm sure we could get rid of it. It does serve to indicate which non-leaf nodes in the menu are clickable (have a content page) and which do not. I'm sure we could do that with color if we wanted (or not do it at all)

File: setup-build-env [hunk 1]
```diff
@@ -0,0 +1,6 @@
+wget https://github.com/gohugoio/hugo/releases/download/v0.80.0/hugo_0.80.0_Linux-64bit.tar.gz
```
Comment (remexre) [inline setup-build-env:1]: These need shebangs!

Comment (remexre) [reply setup-build-env:1]: Also, best to use either wget or curl throughout; some environments don't have one or the other, best to keep deps minimal, etc; I'd just replace lines 1-3 with `curl https://github.com/gohugoio/hugo/releases/download/v0.80.0/hugo_0.80.0_Linux-64bit.tar.gz | tar xv hugo`

Comment (remexre) [reply setup-build-env:1]: alsox2, use `set -e`

**Review by remexre — DISMISSED** [review]

**Comment by remexre** [comment]
also consider merging `typo/propagage` into this; made it against this branch instead of develop since I figure new changes should go here?

**Comment by 602p** [comment]
Yeah, I'll round up all the other feature branches from master into this before it gets merged

**Pull request merged**